Python/tests/datatypes/test_timex_date_helpers.py

```diff
@@ -40,16 +38,18 @@ def test_datatypes_datehelpers_islastweek():
 
 
 def test_datatypes_datehelpers_weekofyear():
-    assert TimexDateHelpers.week_of_year(datetime(2017, 1, 1)) == 1
-    assert TimexDateHelpers.week_of_year(datetime(2017, 1, 2)) == 2
-    assert TimexDateHelpers.week_of_year(datetime(2017, 2, 23)) == 9
-    assert TimexDateHelpers.week_of_year(datetime(2017, 3, 15)) == 12
-    assert TimexDateHelpers.week_of_year(datetime(2017, 9, 25)) == 40
-    assert TimexDateHelpers.week_of_year(datetime(2017, 12, 31)) == 53
-    assert TimexDateHelpers.week_of_year(datetime(2018, 1, 1)) == 1
-    assert TimexDateHelpers.week_of_year(datetime(2018, 1, 2)) == 1
-    assert TimexDateHelpers.week_of_year(datetime(2018, 1, 7)) == 1
-    assert TimexDateHelpers.week_of_year(datetime(2018, 1, 8)) == 2
+    assert TimexDateHelpers.week_of_year(datetime(2017, 1, 1)) == datetime(2017, 1, 1).isocalendar()[1]

```

Comment:
I'd prefer if the right side was kept as numbers as it's easier to visually inspect if values are as expected and to map to other platform tests.

Reply:
Was any incorrect against .NET, for example?